components/telegram_bot_api/actions/delete-message/delete-message.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable camelcase */
+const telegram_bot_api = require("../../telegram_bot_api.app.js");
```

Comment:
Shouldn't all variable names be in lower capital case? like `telegramBotApi`?

Reply:
Some apps like [Microsoft One Drive](https://github.com/PipedreamHQ/pipedream/blob/master/components/microsoft_onedrive/sources/common/base.js) and [Zoho CRM](https://github.com/PipedreamHQ/pipedream/blob/master/components/zoho_crm/sources/common/timer-based/base.js) currently use snake_case rather than camelCase for their app module names, but others like [Google Drive](https://github.com/PipedreamHQ/pipedream/blob/master/components/google_drive/sources/common-webhook.js) and [Google Sheets](https://github.com/PipedreamHQ/pipedream/blob/master/components/google_sheets/sources/common.mjs) use camelCase. It seems like camel case is preferred given the lint rules.

I changed `telegram_bot_api` to `telegramBotApi` and removed `/* eslint-disable camelcase */` from the action files.